tests/extension/oneapi_current_device/current_device.cpp

```diff
@@ -88,10 +88,19 @@ TEST_CASE(
   sycl::device t1_current_device;
   sycl::device t2_current_device;
 
-  std::thread t1(thread_callback, std::cref(t1_device_to_set),
-                 std::ref(t1_current_device));
-  std::thread t2(thread_callback, std::cref(t2_device_to_set),
-                 std::ref(t2_current_device));
+  std::thread t1([&t1_device_to_set, &t1_current_device]() {
```

Comment:
For so simple lambdas and unit-tests, you can deserve some lazyness if you want. :smile:
```suggestion
  std::thread t1([&] {
```

Reply:
This is not a request for a change. @bader this is good to go for me.

Reply:
Thanks.